### DISCUSSION TITLE
did we removed support for webviews?

### DISCUSSION THREAD

**Discussion by akagupta9** [leancodepl/patrol#1226] [Q&A]
My webview interaction is not happening, neither it gives any error.

**Comment by bartekpacia** [comment]
We didn't touch WebView for a few months now. If something worked and suddenly stopped, then it's most likely a bug.

**Comment by akagupta9** [comment]
We are using `flutter_inappwebview` instead of `webview_flutter` package.

with `webview_flutter`  its working. any difference you can see with patrol in terms of these packages?

**Comment by akagupta9** [comment]
any update on it?
